### PR TITLE
setup of backend and frontend

### PR DESCRIPTION
**Pull request by akshithku**
Frontend and Backend setup was done.

### CODE REVIEW

**Review by kutty023 — COMMENTED** [review]
Delete unwanted files

File: Backend/package.json [hunk 1]
```diff
@@ -0,0 +1,5 @@
+{
+  "dependencies": {
+    "i": "^0.3.7"
```
Comment (PatelNeelMahesh) [inline Backend/package.json]: You have added wrong dependency here remove it.

File: frontend/README.md [hunk 2]
```diff
@@ -0,0 +1,70 @@
+# Getting Started with Create React App
```
Comment (PatelNeelMahesh) [inline frontend/README.md]: Remove default Readme generated by create-react-app and add your project component details, figma file etc.

File: frontend/package.json [hunk 3]
```diff
@@ -0,0 +1,38 @@
+{
+  "name": "frontend",
```
Comment (PatelNeelMahesh) [inline frontend/package.json]: Add your project nane here.

**Review by PatelNeelMahesh — CHANGES_REQUESTED** [review]
You have uploaded node_modules in backend, also do the below asked changes.

**Review by PatelNeelMahesh — CHANGES_REQUESTED** [review]
node_module folder is available in PR, delete from here.

**Review by PatelNeelMahesh — APPROVED** [review]
You can merge pull request, after deleting the folder from the PR.

**Pull request merged**